src/components/Profile/ProfileLink/ProfileLink.css

```diff
@@ -3,13 +3,16 @@
   align-items: center;
   gap: 0.5em;
```

Comment:
Note that this gap value is for the spacing between the icon and the text, you might want to reuse this.